tests/e2e/App.spec.ts

```diff
@@ -0,0 +1,306 @@
+import { expect, test } from "@playwright/test";
+
+/**
+  The general shapes of tests in Playwright Test are:
+    1. Navigate to a URL
+    2. Interact with the page
+    3. Assert something about the page against your expectations
+  Look for this pattern in the tests below!
+ */
+
+// If you needed to do something before every test case...
+test.beforeEach(async ({ page }) => {
+  // ... you'd put it here.
+  await page.goto("http://localhost:8000/");
+  await page.getByLabel("Login").click();
+});
+
+// // code below from gearup
+// test("on page load, i dont see the input box until pressing login", async ({
+//   page,
+// }) => {
+//   await expect(page.getByLabel("Login")).toBeVisible();
+//   await expect(page.locator(".repl")).not.toBeVisible();
+// });
+
+// test("after login, the repl is now visible", async ({ page }) => {
+//   await page.getByLabel("Login").click();
+//   await expect(page.locator(".repl")).toBeVisible();
+// });
+
+test("i can switch modes between verbose and brief", async ({ page }) => {
```

Comment:
It could be good to test more interactions between mode and the other commands here, like making sure it actually prints the input for calls to load, view, or search when in verbose mode and that it properly stops printing input for brief mode.